instaclone/src/components/SearchBar/SearchBar.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import instagramLogo from '../../img/instagram-logo.png';
+import instagramWordmark from '../../img/instagram-wordmark.png';
+import PropTypes from 'prop-types';
+
+import './search.scss';
+
+class SearchBar extends React.Component {
+  state = {
+    searchText: ''
+  }
+
+  changeInput = event => {
+    this.setState({
+      searchText: event.target.value
+    }, () => this.props.filterOnChange(event, this.state.searchText))
+    ;
+  }
+
+  logout = event => {
+    event.preventDefault();
+    let userInfoVal = {
+      loggedIn: false,
+      username: '',
+    };
+    localStorage.setItem("instacloneLogin", JSON.stringify(userInfoVal));
```

Comment:
this is a lot of logic for log-out. you could also remove the entire item from localstorage with one command >> `localStorage.removeItem("instacloneLogin")`